<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_write</class>
 <widget class="QDialog" name="d_write">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d_write {
    border-image: url(:/bg/resource/bg_most_2.png);
}
#label{
    color: white;
}
#label_write_Qtitle,#textBrowser_write_Qtext{
    color: #F5EEEE;
}
#pushButton_write_return{
    border-image: url(:/icons/resource/back.png);
}

#pushButton_write_return:hover{
    border-image: url(:/icons/resource/back_h.png);
}
#textBrowser_write_Qtext{
    border: none;
    background-color: rgb(146,168,209);
}

#pushButton_write_ok{
    border-image: url(:/icons/resource/ok.png);
}

#pushButton_write_ok:hover{
    border-image: url(:/icons/resource/ok_h.png);
}

#textEdit_write_Atext{
	border: none;
    background: #F5F1F1
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_write_return">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>4</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_write_Qtext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_write_Atext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>451</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_ok">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_write_Qtitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>49</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回答</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
